Integrations/JiraV2/JiraV2.py

```diff
@@ -305,7 +305,8 @@ def get_issue_fields(issue_creating=False, **issue_args):
     if issue_args.get('summary'):
         issue['fields']['summary'] = issue_args['summary']
 
-    if not issue['fields'].get('project'):
+    # if project key or name is given, add it to the fields, else skip it
+    if (issue_args.get('projectKey') or issue_args.get('projectName')) and not issue['fields'].get('project'):
         issue['fields']['project'] = {}
 
```

Comment:
Hi @avidan-H,
it's because the `jira-edit-issue` does not work at the moment. As there is an empty "project" property added to the request, you will always run in the following error:

![image](https://user-images.githubusercontent.com/1359917/62690841-c40aff00-b9cd-11e9-820b-c3fd51bf676d.png)

Reply:
Wasn't my smartest idea to put this into the same PR.. maybe I should have opened a bug first and put it into a separate PR afterwards 🤔

Reply:
I agree with you that this should be a separate PR 👍 . If you don't mind, it would be much appreciated if you do as you suggested and open a bug and then a separate PR with your fix.

Reply:
Alright. Change has been reverted for this PR. Created #4062 instead